config/changelog.yaml

```diff
@@ -2466,3 +2474,20 @@ Unreleased:
       - Initial release
     wahid:
       - Initial release
+Footer
+� 2022 GitHub, Inc.
+Footer navigation
+
+    Terms
+    Privacy
+    Security
+    Status
+    Docs
+    Contact GitHub
+    Pricing
+    API
+    Training
+    Blog
+    About
+
+You have no unread notifications
```

Comment:
I'm not sure what all this text is about. Was it accidentally added? Or, did one of the lint commands add this text automatically?

Reply:
I have no idea were it came from, so I removed it.